tools/data/hvu/download.py

```diff
@@ -149,7 +149,10 @@ def main(input_csv,
          output_dir,
          trim_format='%06d',
          num_jobs=24,
-         tmp_dir='/tmp/hvu'):
+         tmp_dir='hvu'):
+
+    tmp_dir = os.path.join('/tmp', tmp_dir)
```

Comment:
We should not enforce the tmp_dir to be under the `/tmp/` directory: 
1. What if the user wants to put it in another place, i.e., because little space left in `/`?
2. Poor compatibility: if we want MMAction2 to run on Windows in the future, we need to change here again.

Reply:
`/tmp` -> `./tmp`

Reply:
`tmp_dir = os.path.join(tmp_dir, '.tmp_dir')` now